pipeline/src/pipeline-config/model/pipelineConfig.ts

```diff
@@ -5,7 +5,7 @@ export interface PipelineConfig {
   datasourceId: number
   transformation: TransformationConfig
   metadata: Metadata
-  schema?: object
+  schema: object
```

Comment:
Schould be left optional (same as in adapter) => we will only provide a good query API if schema is given.